public/js/webcam.js

```diff
@@ -29,15 +36,18 @@ async function init() {
 function handleSuccess(stream) {
     window.stream = stream;
     video.srcObject = stream;
+
+    if (video.className === "video-no-animation") {
```

Comment:
Which scenario does this cover?

Reply:
Oh it's like if we click the takePhotoAgain button, just to skip the animation.